fabric/README.md

```diff
@@ -47,26 +47,21 @@ The requirements are:
 * your Service Provider ID (SPID)
 
 In order to use Intel's Attestation Service (IAS) you need to register
-with Intel. [Here](https://software.intel.com/en-us/articles/code-sample-intel-software-guard-extensions-remote-attestation-end-to-end-example)
-you can find more details on how to obtain a signed client certificate,
-registering it and get a SPID.
-
-We currently make use of `unlinkable signatures` for the attestation, thus, when registering with the IAS please choose
-unlinkable signatures. In the case you prefer linkable attestation,
-e.g., because you already have linkable IAS EPID credentials, change
-in line 217 of [../ecc_enclave/sgxcclib/sgxcclib.c](../ecc_enclave/sgxcclib/sgxcclib.c)
-the constant `SGX_UNLINKABLE_SIGNATURE` to `SGX_LINKABLE_SIGNATURE`,
-re-compile and re-deplay [ecc_enclave](../ecc_enclave#build) and [ecc](../ecc#getting-started),
-configure your IAS settings as above with your linkable credentials and run the auction example as follows.
+with Intel. [Here](https://software.intel.com/en-us/form/sgx-onboarding)
```

Comment:
Would it make sense also to link to this here  https://api.portal.trustedservices.intel.com/

Reply:
i thought i fixed that but apparently didn't. anyway, did now (but linked directly to the epid one https://api.portal.trustedservices.intel.com/EPID-attestation as right now we are not yet supporting ecdsa/dcap)

Reply:
mh ... I have registered for a linkable epid. Got a SPID and a primary and a secondary key? which one we I use here?

Reply:
i always use primary, don't know whether there is any difference in secondary, never tried with it.  In any case, updated now the docu to mention to use the primary one (and removed some other now stale info on certs/private-keys which i forgot to fix before ...)